src/ui/wxWidgets/wxUtilities.cpp

```diff
@@ -269,6 +274,31 @@ wxBitmap wxUtilities::GetBitmapResource( const wxString& name )
     wxBitmap bitmap(cpane_xpm);
     return bitmap;
   }
+  else if (name == wxT("graphics/checkmark_placeholder.xpm"))
+  {
+    wxBitmap bitmap(checkmark_placeholder_xpm);
+    return bitmap;
```

Comment:
Nit: we could simplify this to
```suggestion
    return wxBitmap(checkmark_placeholder_xpm);
```

Reply:
Changed as proposed.